odoo/addons/test_lint/tests/test_eslint.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+import re
+import subprocess
+from unittest import skipIf
+from odoo import tools
+
+from . import lint_case
+
+RULES = '{no-undef: error}'
+PARSER_OPTIONS = '{ecmaVersion: 2019, sourceType: module}'
+# Not sure if the global is complete or too big
+GLOBAL = ('owl,openerp,odoo,CKEDITOR,we3,$,jQuery,_,google,window,setTimeout,clearTimeout,'
+    'document,console,QUnit,moment,FileReader,nv,d3,ace,Option,py,XMLHttpRequest,setInterval,'
+    'clearInterval,Image,jstz,ZeroClipboard,sessionStorage,Node,history,gapi,Event,Gravitec,'
+    'navigator,OneSignal,PDFJS,ClipboardJS,PDFSlidesViewer,MutationObserver,Element,URL')
```

Comment:
That makes no sense, most of those globals should be provided explicitely by eslint. And it seems to work fine on my machine, running eslint with `--env browser --env es2017` doesn't err for any of the browser or ES2017 globals.

Reply:
Also would probably be a good idea to define GLOBAL as a joined-list so the maintenance can be less noisy e.g.
```python
GLOBAL = ''.join([
    '_',
    '$', 'jQuery',
    'owl',
    'odoo',
    'google',
    'gapi',
    'ClipboardJS',
    ...
])
```
this way maintenance is just adding / removing entire lines which is much simpler to see especially when the diff doesn't have subline annotations.

Maybe sort them as well? Less sure.